data/global.lua

```diff
@@ -33,6 +33,10 @@ function getFormattedWorldTime()
 	return hours .. ':' .. minutes
 end
 
+function getLootRandom()
```

Comment:
for better randomization you should use `math.randomseed(os.mtime())` before calling `math.random`